### PR TITLE
Test suite modifications

### PR DESCRIPTION
**Pull request by sampath1117**
*Added support to tensor unittests to accept user input for augmentation params

### CODE REVIEW

File: utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp [hunk 1]
```diff
@@ -62,40 +62,150 @@ int main(int argc, char **argv)
...
+    Rpp32f pGridMask[2] = {0.6, 0.5};
+    Rpp32u pGridMaskTranslate[2] = {0, 0};
+
+    //set spatter default values
+    uint pSpatterRGB[3] = {65, 50, 23};
+
     switch (test_case)
     {
     case 0:
         strcpy(funcName, "brightness");
```
Comment (r-abishek) [inline utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp:109]: If the "funcName" comes in as a string from the testAllScript.sh, then this first switch case can be entirely avoided.
You could introduce the default value or the user input value to each param directly inside the second switch case.

File: utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp [hunk 2]
```diff
@@ -62,40 +62,150 @@ int main(int argc, char **argv)
...
+        if(argv[8] != NULL)
+        {
+            pBrightness[0] = atof(argv[8]);
```
Comment (r-abishek) [inline utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp:112]: You may want to add small validator macros or functions (to validate if the user input is in the correct range) in a separate header file and include that here. A common header file that will have helper functions, #defines and includes common to all files.

File: utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp [hunk 3]
```diff
@@ -457,10 +567,11 @@ int main(int argc, char **argv)
 
         Rpp32f alpha[images];
         Rpp32f beta[images];
+
         for (i = 0; i < images; i++)
         {
-            alpha[i] = 1.75;
-            beta[i] = 50;
+            alpha[i] = pBrightness[0];
```
Comment (r-abishek) [inline utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp:573]: After input validation passes, the user param can be used directly. Even if one of the params fails validation, force the user to enter correct params. If no params are entered by the user at all, only then the default params should run.

File: utilities/rpp-unittests/HOST_NEW/testAllScript.sh [hunk 4]
```diff
@@ -154,15 +171,67 @@ rm -rvf ./*
...
+        then
+            varNames="ColorJitter - Brightness, Contrast, Hue, Saturation: "
+        elif [[ case -eq 83 ]]
+        then
+            varNames="GridMask - TileWidth, GridRatio, GridAngle, TranslateX, TranslateY: "
+        elif [[ case -eq 84 ]]
+        then
+            varNames="Spatter - R, G, B: "
+        fi
+        read -p "$varNames" USER_INPUT_ARGUMENTS[CASE_INDEX]
```
Comment (r-abishek) [inline utilities/rpp-unittests/HOST_NEW/testAllScript.sh:214]: Could you confirm if these work - the unit test suite should run in the scenarios:
- running single case - with user params or default values
- running say a couple cases - say 3 cases (./testAllScript 0 2 0 to run case 0, 1, 2) - with user params or default values
- And finally, the all-cases output test where we just run ./testAllScript.sh should run ideally with just the default values like it does now.

**Review by r-abishek — CHANGES_REQUESTED** [review]
Please take a look at these

**Comment by sampath1117** [comment]
Closing the PR since this branch became obsolete
Issued a new PR here
https://github.com/r-abishek/rpp/pull/108

**Pull request closed without merging**